rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -623,35 +623,23 @@ http {
 
     # default server, used for NGINX healthcheck and access to nginx stats
     server {
-        listen {{ $all.ListenPorts.Status }} default_server {{ if $all.Cfg.ReusePort }}reuseport{{ end }} backlog={{ $all.BacklogSize }};
-        {{ if $IsIPV6Enabled }}listen [::]:{{ $all.ListenPorts.Status }} default_server {{ if $all.Cfg.ReusePort }}reuseport{{ end }} backlog={{ $all.BacklogSize }};{{ end }}
-        set $proxy_upstream_name "-";
+        listen unix:/tmp/nginx-status-server.sock;
```

Comment:
Can you avoid hardcoding this here and use https://github.com/kubernetes/ingress-nginx/pull/3684/files#diff-6c2cb34e0bc02a92a477db1ea8101b69R34 instead?

Reply:
done